src/compute/grid/implicit_grid.rs

```diff
@@ -28,12 +28,12 @@ pub(crate) fn compute_grid_size_estimate<'a>(
     // Compute *track* count estimates for each axis from:
     //   - The explicit track counts
     //   - The origin-zero coordinate min and max grid line variables
-    let negative_implicit_inline_tracks = col_min.unsigned_abs();
+    let negative_implicit_inline_tracks = col_min.0.unsigned_abs();
```

Comment:
This diff is an indication that we should be using Deref / DerefMut impls on these newtypes IMO.

Reply:
I'm a bit wary of implementing `Deref`/`DerefMut` for the coordinate newtypes as a large part of the purpose of the types is to make sure that they are not used in the wrong context and I feel like this would weaken that protection. I've taken an alternative approach of:
- Adding methods to `OriginZeroLine`. They still use `.0` internally, but these `.0`'s are now constrained to the impls for that type (i.e they're all `self.0`).
- Just using `OriginZeroLine` directly in more places (avoiding the need to access the inner value entirely).

I've also uncovered a couple of bugs in the process of this, which all seemed to affect grid items that are auto-placed into negative grid tracks. These now have gentests. At some point I would like to go through and convert all the placement unit tests to gentests, but that's definitely for a separate PR.